variables.yml

```diff
@@ -4993,6 +4993,11 @@ Final Energy|Transportation|Waste:
 Final Energy|Waste:
   definition: final energy consumption of non-renewable waste
   unit: PJ/yr
+? Final energy consumed for space heating in residential and commercial buildings
```

Comment:
@danielhuppmann Sieht so aus, als wäre bei der Generierung des yml etwas schiefgelaufen. Hast du eine Idee, was hier passiert sein könnte?

Reply:
Das ist tatsächlich ein valides yaml-Format - der Fehler liegt in Zeile 890 der Excel-Tabelle...